app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -54,6 +53,30 @@ class DrawActivity : AppCompatActivity() {
             it.setOnClickListener { behavior.state = BottomSheetBehavior.STATE_EXPANDED }
         }
 
+        dialogUtils.colorPickerCalculations(bottomSheet = bottomSheet,
+            behavior = behavior,
+            lastHsv = Hsv(0f, 0f, 0f),
+            onColorAdjust = { hsv -> hsv?.let { viewModel.userUpdatesColor(hsv) } },
+            onSubmit = { hsv ->
+                hsv?.let {
+                    viewModel.userUpdatesColor(hsv)
+                    viewModel.userConfirmsColor()
+                }
+            })
+
+        currentColorSelection.setOnClickListener {
+            dialogUtils.colorPickerCalculations(bottomSheet = bottomSheet,
+                behavior = behavior,
+                lastHsv = Hsv(0f, 0f, 0f),
```

Comment:
Make 0f be the default for these fields

Reply:
set to 0f, 1f, 1f since that is solid red, the same default as the gradient.